extensions/amp-auto-ads/0.1/anchor-ad-strategy.js

```diff
@@ -78,7 +79,27 @@ export class AnchorAdStrategy {
       return false;
     }
     for (let i = 0; i < optInStatus.length; i++) {
```

Comment:
Is it worth verifying optInStatus is an array?  I realize this wasn't happening previously... 

Suggested ES6 version of this code:
return optInStatus.some((status) => status == OPT_IN_STATUS_ANCHOR_ADS || status == OPT_IN_STATUS_NO_FILL_ANCHOR_ADS);

Reply:
Done.